community/kernel/src/main/java/org/neo4j/kernel/impl/api/operations/SchemaReadOperations.java

```diff
@@ -89,8 +86,7 @@ PopulationProgress indexGetPopulationProgress( KernelStatement state, NewIndexDe
     String indexGetFailure( Statement state, NewIndexDescriptor descriptor ) throws IndexNotFoundKernelException;
 
     /**
-     * Get all constraints applicable to label and propertyKeys. There are only {@link NodePropertyConstraint}
-     * for the time being.
+     * Get all constraints applicable to label and propertyKeys.
```

Comment:
I guess that since the type is `SchemaDescriptor`, this could be used to fetch rel-type-property-existence-constraints as well.